test/Microsoft.Health.Fhir.Shared.Tests.E2E/Rest/ConvertDataTests.cs

```diff
@@ -135,9 +135,31 @@ public async Task GivenAValidRequest_ButTemplateReferenceIsInvalid_WhenConvertDa
         [Theory]
         [InlineData("fakeacr.azurecr.io/template:default")]
         [InlineData("test.azurecr-test.io/template:default")]
-        [InlineData("test.azurecr.com/template@sha256:592535ef52d742f81e35f4d87b43d9b535ed56cf58c90a14fc5fd7ea0fbb8696")]
         [InlineData("*****####.com/template:default")]
         [InlineData("¶Š™œãý£¾.com/template:default")]
+        public async Task GivenAValidRequest_WithoutDigest_WhenConvertData_ShouldReturnBadRequest(string templateReference)
+        {
+            if (!_convertDataEnabled)
+            {
+                return;
```

Comment:
We can now use [SkippableTheory/Facts](https://github.com/AArnott/Xunit.SkippableFact/blob/master/test/Xunit.SkippableFact.Tests/SampleTests.cs#L51)

Reply:
Awesome! Thanks!